runtime/development/src/lib.rs

```diff
@@ -1844,33 +1844,7 @@ impl pallet_transfer_allowlist::Config for Runtime {
 }
 
 parameter_types! {
-		pub const OrderPairVecSize: u32 = 1_000_000u32;
-}
-
-// Minimum order amounts for orderbook orders v1 implementation.
-// This will be replaced by runtime specifiable minimum,
-// which will likely be set by governance.
-const DEV_USDT_CURRENCY_ID: CurrencyId = CurrencyId::ForeignAsset(1);
-const DEV_AUSD_CURRENCY_ID: CurrencyId = CurrencyId::ForeignAsset(2);
-const DEV_USDT_DECIMALS: u128 = 1_000_000;
-const DEV_AUSD_DECIMALS: u128 = 1_000_000_000_000;
-const DEFAULT_DEV_MIN_ORDER: u128 = 5;
-const MIN_DEV_USDT_ORDER: u128 = DEFAULT_DEV_MIN_ORDER * DEV_USDT_DECIMALS;
-const MIN_DEV_AUSD_ORDER: u128 = DEFAULT_DEV_MIN_ORDER * DEV_AUSD_DECIMALS;
-const MIN_DEV_NATIVE_ORDER: u128 = DEFAULT_DEV_MIN_ORDER * CFG;
-
-parameter_type_with_key! {
-		pub MinimumOrderAmount: |pair: (CurrencyId, CurrencyId)| -> Option<Balance> {
```

Comment:
Where was `MinimumOrderAmount` being used? Will check the code but leaving the code here 👀

Reply:
The pallet now supports admin actions that allow to set those values.